src/lib/substituteResponsiveAtRules.js

```diff
@@ -29,7 +29,11 @@ export default function(config) {
           return cloned
         })
       )
-      css.append(mediaQuery)
+
+      if(mediaQuery.nodes.length) {
+        // do not add media query without any rules or comments
```

Comment:
Minor style changes but can you add a space after the `if`, and remove the comment? I think the code is expressive enough to me without it 👍

Reply:
done